utils/actions/detectConsoleLogs.ts

```diff
@@ -122,19 +142,47 @@ export default async function detectConsoleLogs({
           ],
         })
         .then((result) => {
-          const addtionsHaveConsoleLog = result.data.choices[0].message.content;
+          const openAIResult = result.data.choices[0].message.content.split(",");
+
+          const addtionsHaveConsoleLog = openAIResult[0];
+          const individualLine = openAIResult[1];
 
           if (addtionsHaveConsoleLog === "true") {
-            // comment detailing which file has the console log detected
-            return octokit.request(
-              "POST /repos/{owner}/{repo}/issues/{issue_number}/comments",
-              {
-                owner,
-                repo,
-                issue_number,
-                body: `This PR contains console logs in file ${file.filename}. Please remove them.`,
-              }
-            );
+            const commentFileDiff = () => {
+              return octokit
+                .request("GET /repos/{owner}/{repo}/pulls/{pull_number}", {
+                  owner,
+                  repo,
+                  pull_number: issue_number,
+                })
+                .then((result) => {
+                  const latestCommitHash = result.data.head.sha;
+
+                  return octokit.request(
+                    "POST /repos/{owner}/{repo}/pulls/{pull_number}/reviews",
+                    {
+                      owner,
+                      repo,
+                      pull_number: issue_number,
+                      commit_id: latestCommitHash,
+                      event: "REQUEST_CHANGES",
```

Comment:
Ooof, do we want to request changes? 
If this is the only available option, yes, but we want to give a heads up.
Sometimes I commit console logs to view the prod results!

Reply:
Ok great feedback. We can change the event to `COMMENT`, or `APPROVE`. I'll change it to the former.

Reply:
Thank you!